src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_params.py

```diff
@@ -87,6 +87,20 @@
     c.register_cli_argument('policy assignment create', 'params', options_list=('--params', '-p'),
                             help='JSON formatted string or path to file with parameter values of policy rule')
 
+with VersionConstraint(ResourceType.MGMT_RESOURCE_POLICY, min_api='2017-06-01-preview') as c:
+    existing_policy_set_definition_name_type = CliArgumentType(options_list=('--name', '-n'), completer=get_policy_set_completion_list, help='The policy set definition name')
+    c.register_cli_argument('policy setdefinition', 'policy_set_definition_name', arg_type=existing_policy_set_definition_name_type)
+    c.register_cli_argument('policy setdefinition create', 'name', options_list=('--name', '-n'), help='name of the new policy set definition')
+    c.register_cli_argument('policy setdefinition', 'display_name', help='display name of policy set definition')
+    c.register_cli_argument('policy setdefinition', 'description', help='description of policy set definition')
+    c.register_cli_argument('policy setdefinition', 'definitions',
+                      help='JSON formatted string or a path to a file or uri with such content',
+                      type=file_type, completer=FilesCompleter())
+    c.register_cli_argument('policy assignment create', 'policysetdefinition', options_list=('--policy-set-definition', '-d'),
+                            help='policy set definition name or fully qualified id')
+    c.register_cli_argument('policy assignment create', 'sku', options_list=('--sku', '-s'),
+                            help='policy sku. One of Free or Standard.')
+
```

Comment:
There is no help for `--params` of `az policy setdefinition create` and `az policy setdefinition update`

Reply:
Will add, thanks

Reply:
Done